model/src/main/java/hu/blackbelt/judo/meta/jsl/scoping/JslResourceDescriptionStrategy.xtend

```diff
@@ -127,21 +118,49 @@ class JslResourceDescriptionStrategy extends DefaultResourceDescriptionStrategy
 		val Map<String, String> userData = new HashMap<String, String>
 		 	
 		switch object {
+		 	ModelDeclaration: {
+			 	// System.out.println("(ModeltDeclaration) index:" + object.name)
+
+	 			if (object.name !== null) {
+	 				userData.put("fullyQualifiedName", object.name)
+	 			}
+ 				// System.out.println("Indexing " + object.name)	 			
+	 			if (object.imports !== null) {
```

Comment:
we should check for empty list as well, otherwise we are putting empty string into the "imports"?

Reply:
If empoty, doesn't happen anything, because it is processed with forEach